UI/data/themes/Acri.qss

```diff
@@ -685,6 +685,11 @@ VolumeMeter {
 	qproperty-majorTickColor: rgb(239, 240, 241); /* White */
 	qproperty-minorTickColor: rgb(118, 121, 124); /* Light Gray */
 	qproperty-peakDecayRate: 23.4; /* Override of the standard PPM Type I rate. */
+	qproperty-meterThickness: 3;
+
+	/* The meter numbers use your QWidget or VolumeMeter font, scaled by      */
+	/* meterFontScaling (since qss font-size doesn't support em or %)         */
```

Comment:
QSS = Qt Style Sheet
qss = a file extension

"since the qss file extension font-size doesn't support em or %)" doesn't make sense, so I'd use QSS here.  If you must be more descriptive, then:

> since the QSS font-size property doesn't support em or %